web-embeds/src/index.css

```diff
@@ -38,3 +38,9 @@
 .apollos-widget input:focus-visible {
   outline: none;
 }
+
+@media only screen and (min-device-width: 320px) {
+  #apollos-project-widget > div > div > div > div {
```

Comment:
If the DOM tree changes, will this still work?

Reply:
Moved to more secure place